dct_backend/reviews.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import FastAPI, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+import server
+from fastapi.testclient import TestClient
+
+load_dotenv()
+db = Database(os.getenv("DB_USERNAME"), os.getenv("DB_PASSWORD"))
+db.connect()
+
+food_db = db.client["food"] # from food collection
+review_collection = food_db["reviews"] # review --> sub collection
+app = FastAPI()
+resp = Response
+
+class Reviews():
+    @app.post("/reviews", status_code=status.HTTP_201_CREATED)
+    async def save_review_in_db(review_rating: dict, is_test = False):
+        """
+            if a user has already reviewed an item retun err
+            else insert a review
+
+            *only saves reviews that are not tests
+        """
+        if(not is_test): # if false
+            if review_collection.count_documents({"username": review_rating['username'],"foodID": review_rating['foodID']}, limit = 1) > 0: 
+                # ^^each user can add one review per food item
+                resp.status_code = status.HTTP_400_BAD_REQUEST
+                return {"err": "User has already made a review. Try editing existing review"}
+
+            review_collection.insert_one(review_rating)
+            resp.status_code = status.HTTP_201_CREATED
+            return {"msg": "Review has been successfully created."}
+        
+    
+    @app.post("/reviews", status_code=status.HTTP_201_CREATED)
```

Comment:
You created 2 post methods for the `/reviews` route, which I think technically works, but we don't want to flood that route with 2 different kinds of traffic. Let's create a new route called `/edit-review` maybe?